README.md

```diff
@@ -44,11 +44,15 @@ git clone https://github.com/UXAspects/UXAspects.git
 ```bash
 npm install
 ```
+3. (Optional) Add the webpack-dev-server certificate to the machine's certificate store. This will allow the development server to run using HTTPS without warnings, which is required to test Plunker/CodePen examples locally. On Windows:
```

Comment:
To get this working I had to install openssl on my machine and add it to environment variables. Perhaps this step should be documented?